e2e-tests/steps/main-menu-steps.js

```diff
@@ -59,6 +59,18 @@ export class MainMenuSteps {
         MainMenuSteps.getMenuButton(menuName).click({force: true});
     }
 
+    static clickOnMenuIfNotOpen(menuName) {
```

Comment:
nitpick: this could be just `openMenu` or `expandMenu`

Reply:
it's redundant - removed